sources/se.json

```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "url": "http://slb.nu/slbanalys/luften-idag/",
+        "adapter": "sweden",
+        "name": "Sweden",
+        "city": "Stockkholm",
+        "country": "SE",
+        "description": "Swedish data from SLB analys",
+        "sourceURL": "http://slb.nu/slbanalys/luften-idag/",
+        "contacts": [
+            "info@openaq.org"
```

Comment:
This is currently unused, but may be used to let people know if there are adapter errors in the future. Feel free to put your email in here if you want.